presets/TARGET_DISCO_F769NI/TARGET_BL_INTERNAL_FLASH/mbed_app.json

```diff
@@ -68,6 +69,7 @@
     ],
     "target_overrides": {
         "*": {
+            "target.components_add"                    : ["FLASHIAP"],
```

Comment:
was this actually removed from the target configuration? That would look like a bug.